src/ansys/fluent/core/solver/flobject.py

```diff
@@ -456,10 +457,26 @@ def __getattribute__(self, name):
         if name in super().__getattribute__("child_names"):
             if not self.is_active():
                 raise RuntimeError(f"'{self.path}' is currently not active")
-        return super().__getattribute__(name)
+        try:
+            return super().__getattribute__(name)
+        except AttributeError as ex:
+            attributes = [
+                attribute
+                for attribute in self.__dict__.keys()
+                if not attribute.startswith("_")
+            ]
```

Comment:
@hpohekar looking at the top of this method, it seems that `super().__getattribute__("child_names")` is the relevant list here. If we find ourselves writing code like the above then we need to step back.

Reply:
Done. Thank you.

Reply:
all of the following are same i.e `attributes, self.child_names, super().__getattribute__("child_names")`

![image](https://user-images.githubusercontent.com/106588300/220855144-41d44068-5608-4c1a-8f48-d4d1867a1ca6.png)

`------------------------------------------------------------------`

![image](https://user-images.githubusercontent.com/106588300/220855242-f1f93f4f-aebe-41a2-aac0-27ff283046ff.png)